Mathlib/Order/SuccPred/Limit.lean

```diff
@@ -67,33 +68,74 @@ section Preorder
 
 variable [Preorder α] {a : α}
 
+/-- A successor limit is a value that isn't minimal and doesn't cover any other.
+
+It's so named because in a successor order, a successor limit can't be the successor of anything
+smaller.
+
+This previously allowed the element to be minimal. This usage is now covered by `IsSuccPrelimit`. -/
+def IsSuccLimit (a : α) : Prop :=
+  ¬ IsMin a ∧ IsSuccPrelimit a
+
+protected theorem IsSuccLimit.not_isMin (h : IsSuccLimit a) : ¬ IsMin a := h.1
+protected theorem IsSuccLimit.isSuccPrelimit (h : IsSuccLimit a) : IsSuccPrelimit a := h.2
+
+theorem IsSuccLimit_iff' : IsSuccLimit a ↔ ¬ IsMin a ∧ IsSuccPrelimit a :=
+  Iff.rfl
```

Comment:
Why is this primed. The first letter shouldn't be uppercased. Also do we need this lemma at all?

Reply:
Hm, probably not.